cbauthx/cbauth.go

```diff
@@ -266,7 +266,9 @@ func (a *CbAuth) Reconfigure(config *CbAuthConfig) error {
 	}
 
 	if !a.isCliValidLocked() {
-		a.cli.Close()
+		if a.cli != nil {
```

Comment:
These changes are in both PRs, not sure which one they belong in

Reply:
This change's base-branch is the other one, once it's merged and this PR rebased, those changes should disappear from here.